SS2-Project/Assets/Starstorm2/Modules/Pickups/Items/Lunar/RelicOfEchelon.cs

```diff
@@ -9,17 +9,23 @@
 
 using MSU;
 using MSU.Config;
+using RoR2.ContentManagement;
 
 //Had disabled content, to the nether realm with you. -N
-/*
+
 #if DEBUG
 namespace SS2.Items
 {
-    public sealed class RelicOfEchelon : SS2Item
+    public sealed class RelicOfEchelon : SS2Item, IContentPackModifier
     {
         private const string token = "SS2_ITEM_RELICOFECHELON_DESC";
+        public override ItemDef ItemDef => _itemDef;
+        private ItemDef _itemDef;
+
+        private BuffDef _buffEchelon ;// { get; } = SS2Assets.LoadAsset<BuffDef>("BuffEchelon", SS2Bundle.Indev);
 
-        public override ItemDef ItemDef { get; } = SS2Assets.LoadAsset<ItemDef>("RelicOfEchelon", SS2Bundle.Items);
+        public static Material _overlay;// => SS2Assets.LoadAsset<Material>("matTerminationOverlay");
+        public override NullableRef<List<GameObject>> ItemDisplayPrefabs => throw new NotImplementedException();
```

Comment:
Replace NotImplementedException with null. As a word of thumb, never leave NotImplementedExceptions because an exception will be thrown as soon as its tried to be accessed.